packages/unity-interface/ClientDebug.ts

```diff
@@ -71,6 +71,29 @@ export class ClientDebug {
     this.unityInterface.SendMessageToUnity('Main', 'InstantiateBotsAtCoords', JSON.stringify(payload))
   }
 
+  public StartBotsRandomizedMovement(
+      populationNormalizedPercentage: number,
+      waypointsUpdateTime: number,
+      xCoord: number,
+      yCoord: number,
+      areaWidth: number,
+      areaDepth: number
+  ) {
+    const payload = {
+      populationNormalizedPercentage: populationNormalizedPercentage,
+      waypointsUpdateTime: waypointsUpdateTime,
+      xCoord: xCoord,
+      yCoord: yCoord,
+      areaWidth: areaWidth,
+      areaDepth: areaDepth
+    }
```

Comment:
If you mind, I have a suggestion to make usage more verbosic. since JS has no named parameters, having a long list is hard to track agains a named map

```suggestion
  public StartBotsRandomizedMovement(payload: {
      populationNormalizedPercentage: number,
      waypointsUpdateTime: number,
      xCoord: number,
      yCoord: number,
      areaWidth: number,
      areaDepth: number
  }) {
```